tests/vmi_multus_test.go

```diff
@@ -1084,6 +1102,21 @@ func runSafeCommand(vmi *v1.VirtualMachineInstance, command string) error {
 	}, 15)
 }
 
+func getSRIOVInterfaceNameByMAC(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance, mac string) (string, error) {
+	updatedVmi, err := virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.ObjectMeta.Name, &metav1.GetOptions{})
```

Comment:
I think you better assume you get a refreshed vmi here and not fetch it yourself. Just move it to the caller so others may benefit from it.

Reply:
Done.